v1/ao/internal/reporter/oboe.go

```diff
@@ -292,27 +266,63 @@ func parseInt32(args map[string][]byte, key string, fb int32) int32 {
 	return ret
 }
 
+// mergeLocalSetting follow the predefined precedence to decide which one to
+// pick from: either the local configs or the remote ones, or the combination.
+//
+// Note: This function modifies the argument in place.
+func mergeLocalSetting(remote *oboeSettings) *oboeSettings {
```

Comment:
The precedence is supposed to be:
if both remote (with override flag) and local, lowest setting wins -- lowest is pretty obvious re: sample rate, and we're just saying "and-ed together" for now for the tracing flags. OR
if both remote (without override flag) and local, local setting wins. OR
remote settings